administrator/language/de-DE/com_config.ini

```
@@ -147,7 +147,7 @@ COM_CONFIG_FIELD_METAVERSION_DESC="Steuert, ob der Meta-Tag <code>generator</cod
 COM_CONFIG_FIELD_METAVERSION_LABEL="Joomla-Version anzeigen"
 COM_CONFIG_FIELD_OFFLINE_IMAGE_LABEL="Offline-Bild"
 COM_CONFIG_FIELD_OFFLINE_MESSAGE_LABEL="Eigener Text"
-COM_CONFIG_FIELD_PROXY_ENABLE_DESC="Sorgt dafür, dass Joomla! einen Proxyserver zur Verbindung mit externen URLs nutzt. Dies ist in manchen Serverumgebungen notwendig, damit zum Beispiel die Joomla!-Update-Komponente Daten abrufen kann."
+COM_CONFIG_FIELD_PROXY_ENABLE_DESC="Sorgt dafür, dass Joomla einen Proxyserver zur Verbindung mit externen URLs nutzt. Dies ist in manchen Serverumgebungen notwendig, damit zum Beispiel die Joomla-Update-Komponente Daten abrufen kann."
```

```suggestion
COM_CONFIG_FIELD_PROXY_ENABLE_DESC="Sorgt dafür, dass Joomla! einen Proxyserver zur Verbindung mit externen URLs nutzt. Dies ist in manchen Serverumgebungen notwendig, damit zum Beispiel die Joomla-Update-Komponente Daten abrufen kann."
```
see Comment on line R97